envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/ClusterCircuitBreakerDefaultSettingsTest.kt

```diff
@@ -59,9 +60,11 @@ internal class ClusterCircuitBreakerDefaultSettingsTest {
         // when
         val maxRequestsSetting = envoy.container.admin().circuitBreakerSetting("echo", "max_requests", "default_priority")
         val maxRetriesSetting = envoy.container.admin().circuitBreakerSetting("echo", "max_retries", "high_priority")
+        val remainingSetting = envoy.container.admin().statValue("cluster.echo.circuit_breakers.default.remaining_pending")
```

Comment:
issue mentions two metrics - remaining_pending and remaining_rq
is remaining_rq also turned on and can be checked in the test?